src/database/schemas.ts

```diff
@@ -171,4 +171,14 @@ export default class Database {
   static isRoutineType(type: string): boolean {
     return type === `function` || type === `procedure`;
   }
+
+  static clearAdvisedIndexes(schema: string, name?: string) {
+    let query = `DELETE FROM QSYS2.SYSIXADV WHERE TABLE_SCHEMA = '${schema}'`;
```

Comment:
Just a note here that when IASP support comes in, the catalog schema won't be `QSYS2`...
`// If connected to an IASP we need to query the IASP catalog view`
`return asp > 32 ? QSYS2 + String.format("%05d", asp) : QSYS2;`

Reply:
Interesting.. we have ASP support in the next release. Gotta look into this one more. Thanks!